assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -879,6 +879,13 @@ export default Component.extend({
     this.set("sendingloading", true);
     this._setDraftForChannel(null);
 
+    // TODO: all send message logic is due for massive refactoring
+    // This is all the possible case Im currently aware of
+    // - answering to a public channel where you are not a member yet (preview = true)
```

Comment:
Nitpicky, but I think we should refer to `posting` or `messaging` in this comment block (vs. `answering`).

Reply:
funny I just said in call we should adopt common vocabulary :p  right will change